testing/osde2e/gpu-addon.sh

```diff
@@ -0,0 +1,37 @@
+
+#!/bin/sh
```

Comment:
This will not work unless `#!` are the very first characters in the file on the first line

Reply:
Correct, fixed